modules/tinymce/src/plugins/insertdatetime/test/ts/browser/InsertDatetimeSanityTest.ts

```diff
@@ -1,43 +1,38 @@
-import { ApproxStructure, Log, Pipeline } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { TinyApis, TinyLoader, TinyUi } from '@ephox/mcagar';
+import { ApproxStructure } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinyUiActions } from '@ephox/mcagar';
+import Editor from 'tinymce/core/api/Editor';
 
-import InsertDatetimePlugin from 'tinymce/plugins/insertdatetime/Plugin';
-import SilverTheme from 'tinymce/themes/silver/Theme';
+import Plugin from 'tinymce/plugins/insertdatetime/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
```

Comment:
The `Editor` import is in the wrong location here, as it should be alongside the `Plugin`/`Theme` imports. eg:
```
import { TinyAssertions, TinyHooks, TinyUiActions } from '@ephox/mcagar';

import Editor from 'tinymce/core/api/Editor';
import Plugin from 'tinymce/plugins/insertdatetime/Plugin';
import Theme from 'tinymce/themes/silver/Theme';
```

- [ ] Fix imports